dev-tools/omdb/src/bin/omdb/nexus.rs

```diff
@@ -2623,6 +2625,35 @@ fn print_task_support_bundle_collector(details: &serde_json::Value) {
                 println!(
                     "      Bundle was able to list service processors: {listed_sps}"
                 );
+
+                #[derive(Tabled)]
+                #[tabled(rename_all = "SCREAMING_SNAKE_CASE")]
+                struct StepRow {
+                    step_name: String,
+                    start_time: String,
+                    duration: String,
+                    status: String,
+                }
+
+                steps.sort_unstable_by_key(|s| s.start);
+                let rows: Vec<StepRow> = steps
+                    .into_iter()
+                    .map(|step| {
+                        let duration = (step.end - step.start)
+                            .to_std()
+                            .unwrap_or(Duration::from_millis(0));
+                        StepRow {
+                            step_name: step.name,
+                            start_time: step.start.to_rfc3339(),
+                            duration: format!("{:.3}s", duration.as_secs_f64()),
+                            status: step.status.to_string(),
```

Comment:
since `status` is `Display`, we could just make the `status` field that type instead of ToStringing it...

Reply:
Updated to use stronger type here